src/Lykke.Service.History/Workflow/Projections/TransactionHashProjection.cs

```diff
@@ -87,5 +87,31 @@ public async Task<CommandHandlingResult> Handle(CashoutCompletedEvent @event)
 
             return CommandHandlingResult.Ok();
         }
+
+        /// <summary>
+        ///     BIL batched cashout event
+        /// </summary>
+        /// <param name="event"></param>
+        /// <returns></returns>
+        public async Task<CommandHandlingResult> Handle(
+            Job.BlockchainCashoutProcessor.Contract.Events.CashoutsBatchCompletedEvent @event)
+        {
+            if (@event.Cashouts == null || @event.Cashouts.Length == 0)
+            {
+                _logger.Warning($"BIL batched cashout event, BatchId {@event.BatchId}", context: @event);
```

Comment:
so if Cashouts == null - NRE is throwed below in "foreach" clause. We should add "return" state after line with logging.

Reply:
thx!